packages/core/type-definitions/combinator/timeslice.d.ts

```diff
@@ -1,10 +1,10 @@
 import { Stream } from '@most/types';
 
 export function until<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
-export function until<A>(signal: Stream<A>): (s: Stream<A>) => Stream<A>;
+export function until<A>(signal: Stream<any>): (s: Stream<A>) => Stream<A>;
```

Comment:
It takes a `signal` then returns a function that takes a `stream` returning a new stream that will end when the signal emits.